pmm-app/src/shared/components/helpers/selectThemeVariant.ts

```diff
@@ -0,0 +1,40 @@
+import { selectThemeVariant } from '@grafana/ui';
```

Comment:
This is premature to put it in `/shared`.

Reply:
Idea is to use it later in other panels so I would leave it here